Consul/Client_Options.cs

```diff
@@ -32,26 +33,79 @@ public class QueryOptions
         public static readonly QueryOptions Default = new QueryOptions()
         {
             Consistency = ConsistencyMode.Default,
-            Datacenter = string.Empty,
+            // Datacenter = string.Empty,
```

Comment:
What is the reason for commenting it out, did it cause any troubles?

Reply:
Thanks for spotting this. I missed this. This shouldn't have been commented out.